pgx/src/spi.rs

```diff
@@ -131,7 +131,22 @@ pub enum Error {
 pub struct Spi;
 
 // TODO: should `'conn` be invariant?
-pub struct SpiClient<'conn>(PhantomData<&'conn SpiConnection>);
+pub struct SpiClient<'conn> {
+    phantom: PhantomData<&'conn SpiConnection>,
```

Comment:
oh.  We typically name this kind of member `__marker`.  You'll see that in a bunch of places in pgx.  I think a few stray `_phantom`s snuck in, but `__marker` is what I prefer.  Please rename.

We also typically make it the last member in the struct.

Reply:
Done